server/services/zwavejs-ui/lib/zwaveJSUI.scan.js

```diff
@@ -0,0 +1,14 @@
+const logger = require('../../../utils/logger');
+
+/**
+ * @description This will discovery Z-Wave JS UI devices.
+ * @example zwaveJSUI.scan();
+ */
+async function scan() {
+  logger.info('Asking ZWave JS UI for the list of devices');
```

Comment:
Maybe you should store the "scanning" status, to be able to display "loading" on UI event if we change or refresh page.